shared/pages/PartialClosure/PartialClosure.js

```diff
@@ -390,39 +432,44 @@ export default class PartialClosure extends Component {
               selectedValue={getCurrency}
               onSelect={this.handleSetGetValue}
               label="You buy"
+              tooltip="The amount you receive after the swap"
               disabled
               currencies={currencies}
               usd={getUsd}
             />
             {
-              isSearching && (
+              isSearching || (isNonOffers && maxAmount === 0) && (
                 <span>
                   {` Wait search orders: `}
                   <InlineLoader />
                 </span>
               )
             }
-            <p>{`Max amount for offer:`} {maxAmount}{' '}{getCurrency.toUpperCase()}</p>
-            {maxAmount > 0 && isNonOffers && (
-              <p styleName="error">
-                {`No orders found, try to reduce the amount`}
+            { oneCryptoCost.isGreaterThan(0) && oneCryptoCost.isFinite() && (
+              <p>
+                <FormattedMessage id="PartialPrice" defaultMessage="Price: " />
```

Comment:
Можно в одну строку:
```suggestion
                <FormattedMessage id="PartialPrice" defaultMessage={`Price: 1 ${getCurrency.toUpperCase()} = ${oneCryptoCost.toFixed(5)} ${haveCurrency.toUpperCase()}`} />
```